### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by marcoow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,60 +34,55 @@
...
     "body-parser": "^1.17.1",
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-merge-trees": "^2.0.0",
-    "broccoli-yuidoc": "~2.1.0",
+    "broccoli-yuidoc": "^2.1.0",
     "chai": "^3.5.0",
-    "cors": "~2.8.1",
+    "cors": "^2.8.1",
     "ember-ajax": "^2.4.1",
-    "ember-browserify": "1.1.13",
```
Comment (Turbo87) [inline package.json]: not needed anymore?

File: package.json [hunk 2]
```diff
@@ -34,60 +34,55 @@
...
-    "ember-cli": "2.11.0",
-    "ember-cli-addon-tests": "^0.6.0",
-    "ember-cli-app-version": "^2.0.0",
+    "ember-cli": "2.12.0",
```
Comment (Turbo87) [inline package.json]: most recent is `2.12.1`

File: package.json [hunk 3]
```diff
@@ -34,60 +34,55 @@
   "devDependencies": {
     "body-parser": "^1.17.1",
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-merge-trees": "^2.0.0",
```
Comment (Turbo87) [inline package.json]: not needed?

File: package.json [hunk 4]
```diff
@@ -34,60 +34,55 @@
...
-    "ember-export-application-global": "^1.0.5",
-    "ember-load-initializers": "^0.6.3",
-    "ember-resolver": "^2.0.3",
-    "ember-sinon": "~0.6.0",
-    "ember-source": "^2.11.0",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^3.0.0",
+    "ember-source": "~2.12.0",
+    "ember-sinon": "^0.6.0",
```
Comment (Turbo87) [inline package.json]: `sinon` should be above `source` to match alphabetic sort order

File: package.json (hunk 3 above)
Comment (marcoow) [inline package.json]: I think this was a dependency that Ember CLI used to add? I didn't find any usage in ESA for this.

File: package.json (hunk 1 above)
Comment (marcoow) [inline package.json]: Doesn't look like it - couldn't find any usage.

File: package.json (hunk 3 above)
Comment (Turbo87) [inline package.json]: nope, it's usually used to adjust the build pipeline, but I'm not aware that Ember CLI ever added that as a project dependency

Comment (marcoow) [inline package.json]: Hm, maybe old ESA dependency, pretty sure it's not needed though. Will check again.

Comment (marcoow) [inline package.json]: No, don't need it - should be safe to remove.

**Pull request merged**